packages/react-components/react-avatar/src/stories/AvatarGroup/AvatarGroupSizeSpread.stories.tsx

```diff
@@ -1,6 +1,12 @@
 import * as React from 'react';
-import { AvatarGroup, AvatarGroupItem } from '@fluentui/react-avatar';
+import {
+  AvatarGroup,
+  AvatarGroupItem,
+  AvatarGroupOverflow,
+  getPartitionedAvatarGroupItems,
+} from '@fluentui/react-avatar';
```

Comment:
AFAIK as should be imported from `react-components` now

Reply:
Since AvatarGroup is in unstable, this has to be imported from `@fluentui/react-{component}`. Once it's moved to stable, it will be updated to `@fluentui/react-components`.